packages/datatrak-web-server/src/routes/UserRoute.ts

```diff
@@ -50,7 +50,7 @@ export class UserRoute extends Route<UserRequest> {
       project = projects.find((p: WebServerProjectRequest.ResBody) => p.id === projectId);
     }
     if (countryId) {
-      const countryResponse = await ctx.services.central.fetchResources(`/entities/${countryId}`, {
+      const countryResponse = await ctx.services.central.fetchResources(`entities/${countryId}`, {
```

Comment:
This is unrelated, and it doesn't change behaviour. It just removes a double `/` from the url which was bothering me.